orchestrator/orchestrator_test.go

```diff
@@ -20,45 +17,121 @@ type TestSuite struct {
 	orchestrator *Orchestrator
 }
 
-func createTestSuite(t *testing.T) *TestSuite {
+type TestCase struct {
+	name    string
+	l1Host  string
+	l2Host  string
+	wantErr bool
+}
+
+func createTestSuite(t *testing.T, tc *TestCase) *TestSuite {
```

Comment:
think it will be more flexible to just pass it a networkConfig as a param

```
func createTestSuite(t *testing.T, networkConfig *NetworkConfig) *TestSuite {
```